instrumentation/kafka-clients/README.md

```diff
@@ -103,6 +102,37 @@ to trace manually or you can do similar via automatic instrumentation like Aspec
 }
 ```
 
+## Single Root Span on Consumer
+
+When a Tracing Kafka Consumer is processing records that do not have trace-context (i.e. Producer is not tracing)
+it will reuse the same root span `poll` to group all processing of records returned.
+
+```
+trace 1:
+poll
+|- processing1
+|- processing2
+...
++- processing N
+```
+
+If this is not the desired behavior, users can customize it by setting `singleRootSpanOnReceiveBatch` to `false`. 
+This will create a root span `poll` for each record received. 
+
+```
+trace 1:
+poll
```

Comment:
should this not be `poll1` , `poll2` etc ?

Reply:
IMO it's fine as is, the span name is always `poll` (at least I think so ;) ).

Reply:
yeap it is, that's why :)